core/station/impl/src/errors/system.rs

```diff
@@ -10,6 +10,8 @@ pub enum SystemError {
     InitFailed { reason: String },
     #[error(r#"The canister needs at least one admin"#)]
     NoAdminsSpecified,
+    #[error(r#"There are too many admins defined, max is 255."#)]
```

Comment:
> max is 255

the code checks for u16::MAX which is ~65k

Reply:
i added the max as prop there, this way it's we defined closer to the check